sdk/textanalytics/ai-text-analytics/test/public/textAnalyticsClient.spec.ts

```diff
@@ -479,15 +478,6 @@ describe("[AAD] TextAnalyticsClient", function(this: Suite) {
         assert.equal(result.error.code, "UnsupportedLanguageCode");
       });
 
-      it("service reports warning for long words", async function() {
```

Comment:
nit, you can always set the model version to `2020-07-01` so you can still get this behavior